### DISCUSSION TITLE
What gui library is used ?

### DISCUSSION THREAD

**Discussion by erf** [zed-industries/zed#6550] [Feature Requests]
I like the minimal and responsive GUI in zed. Can you tell me what GUI library (if any) you have used?

**Comment by JosephTLyons** [comment]
We built a custom framework called GPUI.  You can read about it here:

- https://zed.dev/blog/videogame

GPUI is currently closed source and directly tied into the Zed codebase, but we plan to separate it out and open source it at some point.

**Reply by fbreuer** [reply]
Really looking forward to GPUI being open source. Also, great blog post btw.
